tests/test_data/matsim/network.xml

```diff
@@ -15,7 +15,7 @@
 			<attributes>
 				<attribute name="osm:way:access" class="java.lang.String" >permissive</attribute>
 				<attribute name="osm:way:highway" class="java.lang.String" >unclassified</attribute>
-				<attribute name="osm:way:id" class="java.lang.Long" >26997928</attribute>
+				<attribute name="osm:way:id" class="java.lang.Float" >26997928.0</attribute>
```

Comment:
There is some potential inconsistency in the "round trip", where we read in a network with some attributes typed as `java.lang.Long`, they become Python floats, and then if we write them out again they will be, I think, given a `class` of `java.lang.Float`. Do I have that right?

I am struggling to reason about this - could that change cause any problems? I guess that both `java.lang.Long` and `java.lang.Float` become floats in Python anyway, so as far as GeNet is concerned, there is no difference, but could MATSim's behaviour be effected somehow by the difference in type?